osu.Game/Screens/OsuScreen.cs

```diff
@@ -179,6 +179,15 @@ private void updateActivity()
                 api.Activity.Value = activity;
         }
 
+        protected void ResumeIfNoUserPauseRequested()
```

Comment:
I would probably call this something like `ResumeTrackIfRequired()`, in case you need to add different logic in the future.

Reply:
move this logic completely inside `MusicController`. there's no reason for it to be here.

Reply:
Understood